doc/api/deprecations.md

```diff
@@ -2518,6 +2518,27 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`] and
 [`request.socket`][] over [`request.connection`].
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: http finished
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28679
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+[`response.finished`][] indicates whether [`response.end()`] has been
+called and [`response.writableEnded`][] is `true`, not whether the underlying
+data has been flushed, `'finish'` has been emitted and
+[`response.writableFinished`][] is `true`.
+
+Use [`response.writableFinished`][] or [`response.writableEnded`][]
+accordingly instead to avoid the ambigiuty.
+
+To maintain existing functionality use [`response.writableEnded`][].
```

Comment:
I am not absolutely sure what functionality should be maintained here.

Reply:
`finished`

Reply:
That sounds clearer to me than the current wording. I would just use that instead?